CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v1225
```

Comment:
Whenever you get the chance & update this PR with latest main, and before you merge this one, could you remove this from changelog (on `main`):
```
### Some clean up of deprecated code
Some older IE-related code is removed, and so is a query param in our page view RUM call that's no longer being used.
```

Thanks